firefox-ios/Client/Protocols/CanRemoveQuickActionBookmark.swift

```diff
@@ -8,7 +8,8 @@ import Storage
 import class MozillaAppServices.BookmarkItemData
 
 protocol CanRemoveQuickActionBookmark {
-    var bookmarksHandler: BookmarksHandler { get }
+    @MainActor var bookmarksHandler: BookmarksHandler { get }
+    @MainActor
     func removeBookmarkShortcut(quickAction: QuickActions)
```

Comment:
nit; spacing + I think the `@MainActor` can be at the top of that `var`?
```suggestion
    @MainActor 
    var bookmarksHandler: BookmarksHandler { get }
    
    @MainActor
    func removeBookmarkShortcut(quickAction: QuickActions)
```

Reply:
A lot of the protocols i have been working in have no spacing between protocol values. Do we have a team preferences one way or another?